java/oauth-custom-challengehandler/README.md

```diff
@@ -0,0 +1,31 @@
+# Authenticate with OAuth
```

Comment:
Should this say?: "Authenticate with OAuth using a custom AuthenticationChallengeHandler"

Reply:
Updated